src/main/java/Jake.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+public class Jake {
+
+    // List out all tasks
+    private static void listTask(Task[] commands, int totalCommands) {
+        System.out.println("Current commands being executed: ");
+        for (int i=0; i<totalCommands; i++){
+            System.out.println(Integer.toString(i+1) + "." + commands[i].getStatus());
+        }
+    }
```

Comment:
Perhaps the variable name and print statement here can be more specific? Instead of `totalCommands`, I suggest totalTasks as commands such as `mark` and `unmark` are not stored in the array.